lib/ecto_state_machine.ex

```diff
@@ -29,6 +29,20 @@ defmodule EctoStateMachine do
         unquote(events) |> Enum.map(fn(x) -> x[:name] end)
       end
 
+      def unquote(:"get_event_to")(event) do
```

Comment:
2. I dislike current interface. It should show not the direction from `nil` or `default` state, it should accept destination state (or sturct contains this state) and final state.

Result should looks like:

``` elixir
User.get_events_to(:unconfirmed, :confirmed)
User.get_events_to(%User{ state: :unconfirmed }, :confirmed)
```